pkg/resource/deploy/builtins.go

```diff
@@ -142,20 +142,29 @@ func (p *builtinProvider) Read(urn resource.URN, id resource.ID,
 	}, resource.StatusOK, nil
 }
 
+const readStackOutputs = "pulumi:pulumi:readStackOutputs"
 const readStackResourceOutputs = "pulumi:pulumi:readStackResourceOutputs"
 
 func (p *builtinProvider) Invoke(tok tokens.ModuleMember,
 	args resource.PropertyMap) (resource.PropertyMap, []plugin.CheckFailure, error) {
-	if tok != readStackResourceOutputs {
-		return nil, nil, errors.Errorf("unrecognized function name: '%v'", tok)
-	}
 
-	outs, err := p.readStackResourceOutputs(args)
-	if err != nil {
-		return nil, nil, err
-	}
+	if tok == readStackOutputs {
+		outs, err := p.readStackReference(args)
+		if err != nil {
+			return nil, nil, err
+		}
 
-	return outs, nil, nil
+		return outs, nil, nil
+	} else if tok == readStackResourceOutputs {
+		outs, err := p.readStackResourceOutputs(args)
```

Comment:
i was incorrect @pgavlin we had an invoke call for "read stack **resource** outputs", something different from "read stack outputs".  I've now added the latter.